README.md

```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
 
 [![Deploy to Netlify from fork](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/dolbyio-samples/comms-app-react-events)
 
-## Running locally
+## Getting Started
 
-Please see [the events package readme](./events/README.md) for information on how to run the app locally.
+The following steps will quickly get you started testing the Dolby.io Communications APIs capabilities.
+
+### Pre-requisites
+
+To get started building this app you will need a Dolby.io account. You will also need the following -
+
+* [NPM v8.11 or higher](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+* [Yarn v1.22.19](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+* [Node v18.0.0 or higher](https://nodejs.org/en/download)
+
+### Basic terminology
+
+As you browse through the source code and documents, you might come across some of these terms
+
+* **A host** is a participant with additional permissions to manage the conference and other participants.
+* **A viewer** is a participant who can only receive video and audio stream from the conference.
+* A **proxy-app-server** is an intermediary API server that communicates with the Dolby.io Communications Platform in order to provide functionality such as RTS/RTMP/HLS streaming or consuming web-hook data. You can see our sample implementation [here](./backend).
```

Comment:
**proxy-app-server** - do we actually use this exact term (hyphens included) anywhere?

Reply:
Nope, I've unhyphenated it. We do refer to Proxy App Server a fair bit.